install/kubernetes/helm/meshery/charts/meshery-operator/templates/deployment.yaml

```diff
@@ -25,12 +25,12 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-        - name: {{ .Chart.Name }}
+        - name: {{ .Values.mesheryOperator.name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
           # TODO: Need to confirm the meshery-operator image version
-          image: "{{ .Values.image.repository }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          image: "{{ .Values.mesheryOperator.image.repository }}"
+          imagePullPolicy: {{ .Values.mesheryOperator.image.pullPolicy }}
           ports:
             - name: http
```

Comment:
I noticed that the `ports` that we have for the `manager` container in this helm chart doesn't match the manifest that `mesheryctl` uses: https://github.com/meshery/meshery-operator/blob/master/config/manifests/default.yaml#L425-L430, probably need to update this to match the YAML that `meshseryctl` uses as well